gdx/src/com/badlogic/gdx/graphics/PixmapIO.java

```diff
@@ -57,11 +57,11 @@ static public Pixmap readCIM (FileHandle file) {
 
 	/** Writes the pixmap as a PNG with compression. See {@link PNG} to configure the compression level, more efficiently flip the
 	 * pixmap vertically, and to write out multiple PNGs with minimal allocation. */

```

Comment:
This piece of javadoc is no longer relevant then.

Reply:
Thank you. I've added new javadoc for the new method and kept the original javadoc in the old one.